t/ui/15-admin-workers.t

```diff
@@ -124,6 +126,8 @@ subtest 'worker overview' => sub {
     # check worker 2
     is($driver->find_element('tr#worker_2 .worker')->get_text(), 'remotehost:1', 'remotehost:1 shown');
     $driver->find_element('tr#worker_2 .help_popover')->click();
+    wait_for_element(selector => '.popover', description => 'worker status popover is displayed');
```

Comment:
you don't need that. find_element will do exactly this. the whole wait_for_ajax is basically obsolete if you do the proper find_element calls.

Reply:
the problem is more that find_element('.popover') will return from selenium back to perl as soon as the element is there, not when it has the proper text.

Reply:
The problem is when I do not have this the tests start to fail. It could very well just be because another call is worth a "sleep 1". This is what I have written in https://github.com/os-autoinst/openQA/pull/2485/commits/140b62505c006feeb3d3fe83c7e209be3e5cf9aa . So how would I be able to busy-loop until the text is there? Use https://github.com/os-autoinst/openQA/pull/2382#discussion_r346173620 ?